tools/analyzer_plugin/lib/src/plugin.dart

```diff
@@ -81,18 +77,16 @@ import 'package:over_react_analyzer_plugin/src/diagnostic/style_value_diagnostic
 import 'package:over_react_analyzer_plugin/src/diagnostic/variadic_children.dart';
 import 'package:over_react_analyzer_plugin/src/diagnostic/variadic_children_with_keys.dart';
 import 'package:over_react_analyzer_plugin/src/diagnostic_contributor.dart';
-import 'package:over_react_analyzer_plugin/src/util/util.dart';
 
-abstract class OverReactAnalyzerPluginBase extends ServerPlugin
-    with
+mixin OverReactAnalyzerPluginBase
```

Comment:
I made this into a mixin so that we could extend from a stubbed version of `ServerPlugin` in tests